LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2019 Xpring Engineering
```

Comment:
Just checking: is that the correct legal name to use?  For Ripple, the legal name is Ripple Labs, Inc but for tm/(c) purposes, we need to use just "Ripple": 

> © 2013 - 2019 Ripple, All Rights Reserved

Reply:
Follow along here: https://app.asana.com/0/1143722888069470/1144015340821746